CMakeLists.txt

```diff
@@ -408,7 +408,8 @@ if(MSYS OR CYGWIN)
     if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
         set(ENABLE_IPC_CORE FALSE)
     endif()
-    target_compile_definitions(helics_base INTERFACE -D_WIN32_WINNT=0x0601)
+    # setting to windows 10 for all compilation
+    add_compile_definitions(_WIN32_WINNT=0x0A00)
```

Comment:
Does this mean that HELICS will only work on Windows 10 or newer when building under MSYS/Cygwin (e.g. dropping support for older versions)?

Reply:
It means the default is windows 10,  Maybe we want to make a way to override this definition for some odd situations?